src/components/projects/Entry.jsx

```diff
@@ -1,26 +1,64 @@
 import Button from "@material-ui/core/Button";
+import MemberList from "./MemberList";
 import TextField from "@material-ui/core/TextField";
 import { entry as entryValidations } from "./helpers/validations";
+import { postProject } from "./ProjectsAPI";
 import { useFormik } from "formik";
+import { useState } from "react";
+import { useHistory, useLocation } from "react-router-dom";
 import "./Entry.scss";
 
 function Entry() {
+  const [memberList, setMemberList] = useState([]);
+  const history = useHistory();
+  const search = useLocation().search;
+  const idProject = new URLSearchParams(search).get("id");
   const formik = useFormik({
     initialValues: {
-      contact: "",
+      contact: "Jose Ecos",
```

Comment:
Please leave a comment about why we are leaving that initialValue for contact and that it needs to be removed ASAP once the backend is working